apartment_application_service/settings.py

```diff
@@ -38,7 +38,7 @@
     MAILER_LOCK_PATH=(str, ""),
     SENTRY_DSN=(str, ""),
     SENTRY_ENVIRONMENT=(str, ""),
-    LOG_LEVEL=(str, "ERROR"),
```

Comment:
This default is required as otherwise, it fails to error if the log level is not defined.

Reply:
I think the better name for this could be `APPS_LOG_LEVEL` as without prefix `LOG_LEVEL` has a too wide meaning.